### PR TITLE
Reverse a number in c++

### PR DESCRIPTION
**Pull request by tk315**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description
I wrote a code for reverse a number in cpp on article on neoalgo docs which has got merged .
[https://github.com/TesseractCoding/NeoAlgo-Docs/pull/170](https://github.com/TesseractCoding/NeoAlgo-Docs/pull/170)

I would like to add that code here

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

### CODE REVIEW

**Comment by tk315** [comment]
@HarshCasper Please Review

File: C-Plus-Plus/math/reverse_a_number.cpp [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  cout << "Enter the number: ";
+  cin >> num;
+
+  //Printing Output
+
+  cout << "The reverse of the given number is: " << reverse(num);
+  return 0;
+}
+
+"""
```
Comment (atarax665) [inline C-Plus-Plus/math/reverse_a_number.cpp]:
```suggestion
/*
```

File: C-Plus-Plus/math/reverse_a_number.cpp [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+SAMPLE I
+
+INPUT
+Enter the number: 1234
+
+OUTPUT
+The reverse of the given number is: 4321
+
+"""
```
Comment (atarax665) [inline C-Plus-Plus/math/reverse_a_number.cpp]:
```suggestion
*/
```
Add a new line at the EOF.

File: C-Plus-Plus/math/reverse_a_number.cpp [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+int main() {
+  int num;
+
+  //Taking Input 
+
+  cout << "Enter the number: ";
+  cin >> num;
+
+  //Printing Output
+
```
Comment (atarax665) [inline C-Plus-Plus/math/reverse_a_number.cpp:33]: Remove these extra line b/w comment and code

File: C-Plus-Plus/math/reverse_a_number.cpp [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  return 0;
+}
+
+"""
+Time Complexity - O(n), where 'n' is the number of digits in the number.
+Space Complexity - O(1)
+
+SAMPLE INPUT AND OUTPUT
+
+SAMPLE I
```
Comment (atarax665) [inline C-Plus-Plus/math/reverse_a_number.cpp:43]: Remove this line.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/math/reverse_a_number.cpp (hunk 1 above)
Comment (tk315) [reply C-Plus-Plus/math/reverse_a_number.cpp]: done

File: C-Plus-Plus/math/reverse_a_number.cpp (hunk 2 above)
Comment (tk315) [reply C-Plus-Plus/math/reverse_a_number.cpp]: done

File: C-Plus-Plus/math/reverse_a_number.cpp (hunk 3 above)
Comment (tk315) [reply C-Plus-Plus/math/reverse_a_number.cpp:33]: done

File: C-Plus-Plus/math/reverse_a_number.cpp (hunk 4 above)
Comment (tk315) [reply C-Plus-Plus/math/reverse_a_number.cpp:43]: done

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**